devel/getACSBandpass.py

```diff
@@ -20,7 +20,6 @@
 this script, which are the files GALSIM_DIR/share/bandpasses/ACS*.dat, are already included in the
 repository.  This script just lets users know where these files came from and how they were altered.
 """
-from __future__ import print_function
```

Comment:
Probably not.  I'm quite sure some of these devel scripts used GalSim 1.x API, so they won't work.  Some might even predate 1.0 and have additional problems.  Seems like it would be a lot of work to maintain these at even a minimum level of functionality.  I think they are more present just for reference with the expectation that some work might be needed to get them working again if there is a need to run the code again.